Rafael.js

```diff
@@ -0,0 +1 @@
+console.log("We are the champions")
```

Comment:
Bom gosto musical!

Reply:
Politicians hide themselves away
They only started the war
Why should they go out to fight?
They leave that role to the poor, yeah